how-to-guides/fastai-lr-finder.ipynb

```diff
@@ -0,0 +1,436 @@
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "HglaeKrqFlkY"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "source": [
+    "trainer = create_supervised_trainer(model, optimizer, criterion, device=device)\n",
+    "\n",
+    "trainer.run(train_loader, max_epochs=3)"
+   ],
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "State:\n",
+       "\titeration: 1407\n",
+       "\tepoch: 3\n",
+       "\tepoch_length: 469\n",
+       "\tmax_epochs: 3\n",
+       "\toutput: 0.5554001927375793\n",
+       "\tbatch: <class 'list'>\n",
+       "\tmetrics: <class 'dict'>\n",
+       "\tdataloader: <class 'torch.utils.data.dataloader.DataLoader'>\n",
+       "\tseed: <class 'NoneType'>\n",
+       "\ttimes: <class 'dict'>"
+      ]
+     },
+     "metadata": {
+      "tags": []
+     },
+     "execution_count": 24
+    }
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "dKihuuH4A-sH",
+    "outputId": "699eebb6-7446-40cd-9b91-ebdaea07eaeb"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "source": [
+    "evaluator = create_supervised_evaluator(\n",
+    "    model, metrics={\"Accuracy\": Accuracy(), \"Loss\": Loss(criterion)}, device=device\n",
+    ")\n",
+    "evaluator.run(test_loader)\n",
+    "\n",
+    "print(evaluator.state.metrics)"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "{'Accuracy': 0.8655, 'Loss': 0.602867822265625}\n"
+     ]
+    }
+   ],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "mophHZUkICKI",
+    "outputId": "378c7aac-b3c8-49ff-8be1-acc547730c12"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's see how we can achieve better results by using the `FastAILRFinder` handler. But first, let's restore the initial state of the model and optimizer so we can re-train them from scratch. "
+   ],
+   "metadata": {
+    "id": "U_EHmN2bmvd2"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "source": [
+    "model.load_state_dict(init_model_state)\n",
+    "optimizer.load_state_dict(init_opt_state)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "CTGJPVI6mvd2"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "When attached to the `trainer`, this handler follows the same procedure used by [fastai](https://docs.fast.ai/callback.schedule.html#LRFinder). The model is trained for `num_iter` iterations while the learning rate is increased from its initial value specified by the optimizer to the upper bound called `end_lr`. This increase can be linear (`step_mode=\"linear\"`) or exponential (`step_mode=\"exp\"`). The default `step_mode` is exponential which is recommended for larger learning rate ranges while linear provides good results for small ranges."
```

Comment:
> learning rate is increased from its initial value specified by the optimizer to the upper bound called `end_lr`

That's not 100% true anymore, we recently added the start_lr option, see here https://github.com/pytorch/ignite/pull/2111
, so I think saying mentioning it will increase from `start_lr` to `end_lr` and if `start_lr` is None it will start from LR with optimizer be default would be better.

Reply:
Thank you for pointing that out! I'll make the change.